rules/falco_rules.yaml

```diff
@@ -3397,6 +3397,29 @@
   output: "File execution detected from /dev/shm (proc.cmdline=%proc.cmdline connection=%fd.name user.name=%user.name user.loginuid=%user.loginuid container.id=%container.id evt.type=%evt.type evt.res=%evt.res proc.pid=%proc.pid proc.cwd=%proc.cwd proc.ppid=%proc.ppid proc.pcmdline=%proc.pcmdline proc.sid=%proc.sid proc.exepath=%proc.exepath user.uid=%user.uid user.loginname=%user.loginname group.gid=%group.gid group.name=%group.name container.name=%container.name image=%container.image.repository)"
   priority: WARNING
   tags: [host, container, mitre_execution, mitre_defense_evasion, T1036.005, T1059.004]
+  
+- list: other_utilities
+  items: [awk, sed, grep, python, perl, bash, cron, systemd-timer, atd]
+  
+- rule: Execute Command Via Utility
+  desc: Detects execution of commands via utilities commonly used to parse text, scripting languages, and system utilities
+  condition: >
+    (evt.type = execve and
+     (proc.name in (other_utilities) or
+      proc.cmdline contains awk or
+      proc.cmdline contains sed or
+      proc.cmdline contains grep or
+      proc.cmdline contains python or
+      proc.cmdline contains perl or
+      proc.cmdline contains bash or
+      proc.cmdline contains cron or
+      proc.cmdline contains systemd-timer or
+      proc.cmdline contains atd))
```

Comment:
Here we are triggering this rule every time one of these programs is executed, I don't think it is what we want (also by reading the description of T1202). Maybe you want to detect execution of these programs whenever they are not   a child of most used shells? To do that, check out the `proc.aname` field and the `shell_proc_binaries` macro